### PR TITLE
Link fix and feature

### PR DESCRIPTION
**Pull request by mike-nl**

### CODE REVIEW

File: src/selection.js [hunk 1]
```diff
@@ -53,22 +53,42 @@ function Selection(currentElem, nativeSelection) {
...
+    - 'parent'
+    - 'child'
+    - 'same'
+  */
+  function compareWithElement(element, relationship) {
+    if (!native.anchorNode() || !native.focusNode() || !element || !util.isElement(element)) {
       return false;
     } else {
-      var within = false;
+      var result = false;
```
Comment (blambo) [inline src/selection.js:78]: Why are we renaming this from a more specific to less specific variable name?

File: src/selection.js [hunk 2]
```diff
@@ -53,22 +53,42 @@ function Selection(currentElem, nativeSelection) {
     }
   };
 
+  me.getParentElement = function() {
```
Comment (blambo) [inline src/selection.js]: This seems to assume that the selection is all within a single parent, what if the selection is like:
`<p><b>12[34</b>as]df</p>`
Should the parent element be the `<b>` element or the `<p>`?

File: src/toolbar.js [hunk 3]
```diff
@@ -204,6 +216,11 @@ Toolbar.prototype.addDefaultLinkButton = function(label) {
 
         // Add link to selection
         document.execCommand('createLink', false, me.urlTextbox.value);
+
+        if (options.openInNewTab) {
+          // Add target='_blank' to open in a new tab
+          selection.getParentElement().target = '_blank';
```
Comment (blambo) [inline src/toolbar.js]: This makes an assumption about where our selection is after the `execCommand` that I don't think we can necessarily make.

**Review by blambo — CHANGES_REQUESTED** [review]

File: src/selection.js (hunk 1 above)
Comment (mike-nl) [reply src/selection.js:78]: because `within` only makes sense when we were finding whether an element was within another

**Review by blambo — APPROVED** [review]

**Pull request merged**